community/kernel/src/main/java/org/neo4j/unsafe/impl/batchimport/Configuration.java

```diff
@@ -43,7 +43,7 @@
      * database directory of the imported database, i.e. <into>/bad.log.
      */
     String BAD_FILE_NAME = "bad.log";
-    long MAX_PAGE_CACHE_MEMORY = mebiBytes( 240 );
+    long MAX_PAGE_CACHE_MEMORY = mebiBytes( 480 );
```

Comment:
IIRC this is because the cache is only used for writes, and thus there is no benefit of keeping the data in memory.